stable/kafka-pubsub-emulator/templates/statefulset.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1beta2
```

Comment:
Can you please update the `apiVersion` to `apps/v1`. The Kubernetes API will soon stop accepting older versions.